services/history-service/src/entry-points/api/history-routes.ts

```diff
@@ -23,6 +23,13 @@ export default function defineRoutes(expressApp: express.Application) {
   router.get("/", async (req, res, next) => {
```

Comment:
Should we validate the incoming queries to make sure that they are valid, such that otherwise, we return 422 Validation Error to the client

Reply:
Done